source/solvers/navier_stokes_vof_assemblers.cc

```diff
@@ -601,42 +601,46 @@ GLSNavierStokesVOFAssemblerMarangoni<dim>::assemble_rhs(
   // Loop over the quadrature points
   for (unsigned int q = 0; q < n_q_points; ++q)
     {
-      // Gather density
-      double density_eq = scratch_data.density[q];
+      const double &curvature_value = scratch_data.curvature_values[q];
 
-      // Gather filtered phase fraction gradient
-      const Tensor<1, dim> &filtered_phase_fraction_gradient_value =
-        scratch_data.filtered_phase_fraction_gradient_values[q];
+      // Gather phase fraction gradient
+      const Tensor<1, dim> &phase_gradient_value =
+        scratch_data.phase_gradient_values[q];
 
-      const double filtered_phase_fraction_gradient_norm =
-        filtered_phase_fraction_gradient_value.norm();
+      const double phase_gradient_norm = phase_gradient_value.norm();
 
-      const Tensor<1, dim> normalized_filtered_phase_fraction_gradient =
-        filtered_phase_fraction_gradient_value /
-        (filtered_phase_fraction_gradient_norm + DBL_MIN);
+      const Tensor<1, dim> normalized_phase_fraction_gradient =
+        phase_gradient_value / (phase_gradient_norm + DBL_MIN);
+
+      // Gather temperature
+      const double temperature = scratch_data.temperature_values[q];
 
       // Gather temperature gradient
       const Tensor<1, dim> temperature_gradient =
         scratch_data.temperature_gradients[q];
 
       const double JxW_value = JxW[q];
 
+
+      const Tensor<1, dim> tmp_STF =
```

Comment:
I suggest using a more verbose name here.

Reply:
I changed it in GLSNavierStokesVOFAssemblerMarangoni and GLSNavierStokesVOFAssemblerSTF